src/cluster/generated/proto/placementpb/placement.proto

```diff
@@ -40,15 +40,20 @@ message Placement {
 }
 
 message Instance {
-  string id = 1;
-  string isolation_group = 2;
-  string zone = 3;
-  uint32 weight = 4;
-  string endpoint = 5;
-  repeated Shard shards = 6;
-  uint32 shard_set_id = 7;
-  string hostname = 8;
-  uint32 port = 9;
+  string id                 = 1;
+  string isolation_group    = 2;
+  string zone               = 3;
+  uint32 weight             = 4;
+  string endpoint           = 5;
+  repeated Shard shards     = 6;
+  uint32 shard_set_id       = 7;
+  string hostname           = 8;
+  uint32 port               = 9;
+  InstanceMetadata metadata = 10;
```

Comment:
nit: if you did this alignment manually, can you undo it? i don't think prototool does this automatically, but if it did, that's fine!